CablePairTester_CommonCfg.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * CablePairTester_CommonCfg.h
+ *
+ *  Created on: Feb 22, 2021
+ *      Author: PUL1KOR
+ */
+
+#ifndef CABLEPAIRTESTER_COMMONCFG_H_
+#define CABLEPAIRTESTER_COMMONCFG_H_
+
+#include <stdio.h>
+#include <assert.h>
+
+/************
+ * Colour Pair as per standard
+ ************/
+
+enum MajorColor_ten {WHITE, RED, BLACK, YELLOW, VIOLET};
+enum MinorColor_ten {BLUE, ORANGE, GREEN, BROWN, SLATE};
+
+/*************************************/
+
+typedef struct {
+    enum MajorColor_ten majorColor_en;
+    enum MinorColor_ten minorColor_en;
+} ColorPair_tst;

```

Comment:
This is the ColorPair. The _tst convention is unfamiliar. Does it stand for test?
This struct is not only used for testing, it's also used for functionality (like if you print all the color pairs)